packages/js/src/server/integrations/unhandled_rejection_monitor.ts

```diff
@@ -18,13 +18,6 @@ export default class UnhandledRejectionMonitor {
 
   makeListener() {
     const honeybadgerUnhandledRejectionListener = (reason: unknown, _promise: Promise<unknown>) => {
-      if (!this.__client || !this.__client.config.enableUnhandledRejection) {
```

Comment:
Redundant now that the listener is removed in this case. 

I'd only included `!this.__client` here because typescript wouldn't let me check `!this.__client.config.enableUnhandledRejection` otherwise.